src/main/java/org/opensearch/security/securityconf/DynamicConfigFactory.java

```diff
@@ -184,6 +189,7 @@ public void onChange(Map<CType, SecurityDynamicConfiguration<?>> typeToConfig) {
         final WhitelistingSettings whitelist = (WhitelistingSettings) cr.getConfiguration(CType.WHITELIST).getCEntry("config");
         final AllowlistingSettings allowlist = (AllowlistingSettings) cr.getConfiguration(CType.ALLOWLIST).getCEntry("config");
         final AuditConfig audit = (AuditConfig)cr.getConfiguration(CType.AUDIT).getCEntry("config");
+        final TenancyConfigModel tcm ;
```

Comment:
nit: extra space

Reply:
Thanks @shikharj05 for pointing this out. Will fix in next commit.